setup.py

```diff
@@ -39,7 +40,7 @@
     install_requires=['pyephem',
                       'katpoint',
                       'matplotlib<3',
-                      'numpy',
+                      'numpy>=1.17' if sys.version_info >= (3, 5) else 'numpy<1.17',
```

Comment:
Tests are failing for Python 3 as 1.17 isn't available yet.  See comment in katconf PR.